### PR TITLE
Adds support for Play 2.7.x

### PR DESCRIPTION
**Pull request by renatocaval**

### CODE REVIEW

File: src/main/resources/2.7.x.conf [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+      {
+        # Keep the gradle version updated
+        path = "**gradle/wrapper/gradle-wrapper.properties"
+        conversions = {
+          "distributionUrl" = """distributionUrl=https\://services.gradle.org/distributions/gradle-4.9-bin.zip"""
+        }
+      }, 
+
+      {
+        # Keep the gradle version updated
```
Comment (dwijnand) [inline src/main/resources/2.7.x.conf]: stale

File: src/main/resources/2.7.x.conf [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+        path = ".travis.yml"
+        conversions = {
+          "- oraclejdk10" = """- openjdk10"""
+
+           "jdk: oraclejdk10" = """      jdk: openjdk10"""
```
Comment (dwijnand) [inline src/main/resources/2.7.x.conf]: Extra spaces.

Comment (renatocaval) [reply src/main/resources/2.7.x.conf]: this are needed, that's for travis yml file

Comment (dwijnand) [reply src/main/resources/2.7.x.conf]: Oh, so it doesn't honour the spaces that are in front of "jdk: oraclejdk10"?

Comment (renatocaval) [reply src/main/resources/2.7.x.conf]: no, it detects a line containing it an replace the whole line

As I said, TemplateControl deserves some love. Especially, if we move into more automation and want to make release cheaper.

Comment (dwijnand) [reply src/main/resources/2.7.x.conf]: Yeah, sounds like it!

**Comment by dwijnand** [comment]
We can clean up the stale comment after.

**Pull request merged**